test/common/doctest.h

```diff
@@ -1555,10 +1555,13 @@ DOCTEST_CLANG_SUPPRESS_WARNING_WITH_PUSH("-Wunused-comparison")
                 , m_at(at) {}
 
         DOCTEST_NOINLINE operator Result() {
+DOCTEST_GCC_SUPPRESS_WARNING_WITH_PUSH("-Waddress")
+// TODO: upstream the change to doctest : Work-around for the warning: 'address will never be NULL'
```

Comment:
```suggestion
// TODO: upstream the change to doctest : Work-around for the warning: 'address will never be NULL'
DOCTEST_GCC_SUPPRESS_WARNING_WITH_PUSH("-Waddress")
```
Should be this ordering

Reply:
Ok. Fixed it.